docs/workflows/manage-content.rst

```diff
@@ -31,7 +31,18 @@ related to manifests because of that. In pulp, it is required to specify
 a digest of a manifest in order to create a tag for the corresponding
 image.
 
-``http POST http://localhost:24817/pulp/api/v3/docker/tag/ repository=${REPOSITORY_HREF} tag=${TAG_NAME} digest=${MANIFEST_DIGEST}``
+Here is provided an example on how to properly tag an image. First, a
+digest of an eligible manifest is selected. Then, a custom tag is applied
```

Comment:
s/eligible/existing

Reply:
btw, in your script you find the manifest by tag and not by digest, is that intended?

Reply:
Yes, it is. Because I do not know a digest for that particular manifest, or do I?

Reply:
since we are working with the fixtures repo that we have created, we control the content there, so you can look what digest points to manifest_a tag( for example)